sklearn/datasets/_openml.py

```diff
@@ -667,7 +667,7 @@ def fetch_openml(name=None, *, version='active', data_id=None, data_home=None,
         If True, returns ``(data, target)`` instead of a Bunch object. See
         below for more information about the `data` and `target` objects.
 
-    as_frame : bool or 'auto', default=False
+    as_frame : bool or 'auto', default='auto'
```

Comment:
We cannot change the default without a deprecation cycle. It is true that `fetch_openml` is experimental so we could have a single deprecation cycle instead of 2 cycle.

Reply:
You can refer to the documentation: https://scikit-learn.org/stable/developers/contributing.html#deprecation regarding how to deprecate or change the value of a parameter.